timesketch/lib/utils.py

```diff
@@ -37,7 +37,7 @@ def random_color():
     return u'{0:02X}{1:02X}{2:02X}'.format(rgb[0], rgb[1], rgb[2])
 
 
-def read_and_validate_csv(path):
+def read_and_validate_csv(path, delimiter=","):
```

Comment:
Please, use single quotes.  Also, the variable is declared but not used in the function. please make the proper changes to add the variable i.e. 

`reader = csv.DictReader(fh, delimiter=delimiter.decode('string_escape'))`

Reply:
done